cfgov/unprocessed/root/financial-intuition.xml

```diff
@@ -16,6 +16,51 @@
         />
         <itunes:category text="Education"></itunes:category>
         <itunes:explicit>false</itunes:explicit>
+        <item>
+            <title>Understanding your Financial Aid Offer, Part 3: Using the CFPB’s Grad Path Tool to Help with College Decision-making</title>
+            <link>https://www.consumerfinance.gov/consumer-tools/educator-tools/students/financial-intuition/understanding-your-financial-aid-offer-part-3/</link>
+            <itunes:episodeType>full</itunes:episodeType>
+            <itunes:episode>9</itunes:episode>
+            <description>For the third and final episode of this series, we spoke with Kate Mullan, Policy Analyst, Consumer Financial Protection Bureau, to discuss the Grad Path tool and how it can be used to compare programs, assess funding needs (or surplus), and create a financial plan for the academic year.</description>
+            <enclosure
+                length="24859707"
+                type="audio/mpeg"
+                url="https://files.consumerfinance.gov/f/media/cfpb_financial-intuition-podcast_understanding-your-financial-aid-offer_episode-3.mp3"
+            />
+            <pubDate>Mon, 12 Apr 2021 16:00:00 EST</pubDate>
+            <itunes:duration>1553</itunes:duration>
+            <itunes:explicit>false</itunes:explicit>
+        </item>
+        <item>
+            <title>Understanding your Financial Aid Offer, Part 2: What to know when filling out the FAFSA</title>
+            <link>https://www.consumerfinance.gov/consumer-tools/educator-tools/students/financial-intuition/understanding-your-financial-aid-offer-part-2/</link>
+            <itunes:episodeType>full</itunes:episodeType>
+            <itunes:episode>8</itunes:episode>
+            <description>During the second episode in this series, we met with Fred Stennis, Supervisor and Senior Advisor at the U.S. Department of Education, to talk about what to know before, during, and after completing the FAFSA. This episode also covered tips and best practices for using federal financial aid to complete your higher education goals.</description>
+            <enclosure
+                length="18354362"
+                type="audio/mpeg"
+                url="https://files.consumerfinance.gov/f/media/cfpb_financial-intuition-podcast_understanding-your-financial-aid-offer_episode-2.mp3"
+            />
+            <pubDate>Mon, 12 Apr 2021 16:00:00 EST</pubDate>
```

Comment:
For past episodes, we've staggered the timestamps by one second to make sure podcast apps show them in the right order. Also, EDT instead of EST now.

```suggestion
            <pubDate>Mon, 12 Apr 2021 16:00:01 EDT</pubDate>
```